app/routes/user.py

```diff
@@ -121,7 +122,7 @@ def fetch_all():
     })
 
 @userRouter.route('/user/register',methods=["POST"])
-@Middleware.verifyPasswordRules
+@Middleware.validatePassword()
```

Comment:
```suggestion
@Middleware.validatePassword
```
sorry, i forgot to mention, no need for parentheses if arguments aren't passed